### PR TITLE
Sync2 david

### PR DESCRIPTION
**Pull request by purplehays09**
So this has reverted to a previous save like 5 times. I have remade all of the changes and they are running in the system. sorry for the crap ton of changes all at once but I can break them down if you need.

### CODE REVIEW

File: api/cases/cases-model.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+// | Request | URL | Description | Returns | Ready |
+// | ------- | --- | ----------- | ------- | ------- |
+
+// getCases function
+// | GET | api/cases/ | gets all cases | array of all cases | |
+// get all cases, AND tags associated with cases (getTagsByCase)
+// return an array of all cases with an array of all tags
+
+async function getCases() {
```
Comment (andremichalowski) [inline api/cases/cases-model.js:19]: Class work on this David. This must have been a solid amount to work out.

Comment (andremichalowski) [reply api/cases/cases-model.js:19]: Glad you resolved the 'case' issue.

File: api/tags/tags-model.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    getTags,
+    insertTag,
+    getTagById,
+    // updateTag,
+    // removeTag,
+}
+
+async function getTags(){
+    return await db
+        // .join(
```
Comment (andremichalowski) [inline api/tags/tags-model.js]: I don't personally love to do this so i'm just parroting whats been told me but you might want to consider removing commented out code.

File: data/migrations/20200625220949_create-profile.js [hunk 3]
```diff
@@ -1,15 +1,72 @@
 exports.up = (knex) => {
   return knex.schema
-    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+    .raw("CREATE EXTENSION IF NOT EXISTS 'uuid-ossp'")
```
Comment (andremichalowski) [inline data/migrations/20200625220949_create-profile.js]: Passes for me but i'm curious about this let's talk later.

**Review by andremichalowski — APPROVED** [review]
Massive effort David and Sean. Way to go glad to see this wrapping up for you.

**Review by bejolo018 — APPROVED** [review]
Agree with Andre that in the future I would recommend removing commented out code, also probably shouldn't do such large pull requests in the future, other than that everything here lgtm! Great work!!!!!!

**Pull request merged**